package labsession29;

public class Exception1{
	 public static void main(String[] args) {
	        try {
	            System.out.println("Try Block");
	        } catch (Exception e) {
	            System.out.println("Catch Block");
	        } 
	        }
	    }
